server/src/paygov.es6

```diff
@@ -0,0 +1,208 @@
+  const url_cancel = `${vcapConstants.intakeClientBaseUrl}applications/christmas-trees/forests/${forestAbbr}/new`;
+  const xmlTemplate = [
+    {
+      'S:Envelope': [
+        {
+          _attr: {
+            'xmlns:S': 'http://schemas.xmlsoap.org/soap/envelope/'
+          }
+        },
+        {
+          'S:Header': []
+        },
+        {
+          'S:Body': [
+            {
+              'ns2:startOnlineCollection': [
+                {
+                  _attr: {
+                    'xmlns:ns2': 'http://fms.treas.gov/services/tcsonline'
+                  }
+                },
+                {
+                  startOnlineCollectionRequest: [
+                    {
+                      tcs_app_id: tcsAppID
+                    },
+                    {
+                      agency_tracking_id: permit.permitId
+                    },
+                    {
+                      transaction_type: 'Sale'
+                    },
+                    {
+                      transaction_amount: permit.totalCost
+                    },
+                    {
+                      language: 'en'
+                    },
+                    {
+                      url_success: url_success
+                    },
+                    {
+                      url_cancel: url_cancel
+                    },
+                    {
+                      account_holder_name: permit.firstName + ' ' + permit.lastName
+                    },
+                    {
+                      custom_fields: [
+                        {
+                          custom_field_1: orgStructureCode
+                        }
+                      ]
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    }
+  ];
+
+  return xml(xmlTemplate);
+};
+
+/**
+ * Generate XML from the template to use for completing pay.gov transaction.
+ */
+paygov.getXmlToCompleteTransaction = paygovToken => {
+  const xmlTemplate = [
+    {
+      'S:Envelope': [
+        {
+          _attr: {
+            'xmlns:S': 'http://schemas.xmlsoap.org/soap/envelope/'
+          }
+        },
+        {
+          'S:Header': []
+        },
+        {
+          'S:Body': [
+            {
+              'ns2:completeOnlineCollection': [
+                {
+                  _attr: {
+                    'xmlns:ns2': 'http://fms.treas.gov/services/tcsonline'
+                  }
+                },
+                {
+                  completeOnlineCollectionRequest: [
+                    {
+                      tcs_app_id: vcapConstants.payGovAppId
+                    },
+                    {
+                      token: paygovToken
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    }
+  ];
+  return xml(xmlTemplate);
+};
+
+paygov.generateSvgPermit = permit => {
+  return new Promise((resolve, reject) => {
```

Comment:
why is generating the permit in a file called paygov?

Also this method is quite long and i I think we may want to break it out into multiple sections - i.e. rules/ applicantInfo/ and permit static info.

Reply:
I've created a task to rename and break up the method in #168